doc/Language/objects.pod6

```diff
@@ -553,6 +553,13 @@ CATCH { default { put .^name ~ ":\n" ~ .Str } }
 # 'FunMath'. Did you mean '!do-subtraction'?␤»
 =end code
 
+Private methods have their own namespace. Since they're not virtual, missing
```

Comment:
What's this namespace? Is it not `GLOBAL`? Is there a different one per class? What do you mean by "virtual"?

Reply:
I used the information in this [issue](https://github.com/rakudo/rakudo/issues/3801), so hopefully @jnthn can clarify all these questions for us.

Reply:
He clarified a bit further, check it out...

Reply:
>Private methods have their own namespace. They're not virtual, i.e.,
private methods cannot be overriden within the inheriting class to provide any
polymorphic behavior, thus missing ones are detected at compile time...

As for the namespace, I don't know.

Reply:
OK. Please add it to the document.